tests/backup/backup_test.go

```diff
@@ -168,14 +186,35 @@ var _ = Describe("{BasicBackupCreation}", func() {
 			monthly_policy_status := Backupschedulepolicy("monthly", uuid.New(), orgID, monthly_schedule_policy_info)
 			dash.VerifyFatal(monthly_policy_status, nil, "Creating monthly schedule policy")
 		})
+
 		Step("Register cluster for backup", func() {
-			CloudCredUID = uuid.New()
-			CreateCloudCredential("azure", "azureaccount", CloudCredUID, orgID)
-			// To create cloud clustre
-			RegisterBackupCluster(orgID, "azure", CloudCredUID)
+			// To create cloud cluster
+			//CloudCredUID = uuid.New()
```

Comment:
remove the commented code if not required

Reply:
Done